src/react/Router.jsx

```diff
@@ -93,7 +93,7 @@ export default ({ reactHistory: history, astarteClient, sessionManager, config,
         />
         <Route path="groups" element={<GroupsPage {...pageProps} />} />
         <Route path="groups/new" element={<NewGroupPage {...pageProps} />} />
-        <Route path="groups/:groupName" element={<GroupDevicesSubPath {...pageProps} />} />
+        <Route path="groups/:groupName/edit" element={<GroupDevicesSubPath {...pageProps} />} />
```

Comment:
You forgot the changes to the other resources, such as `/devices/:deviceId` -> `/devices/:deviceId/edit`

Reply:
This is just a forward port. Changes to Flow pages have their own PR https://github.com/astarte-platform/astarte-dashboard/pull/271